airflow/www/templates/airflow/tree.html

```diff
@@ -136,7 +136,7 @@
 
 var devicePixelRatio = window.devicePixelRatio || 1;
 // JSON.parse is faster for large payloads than an object literal (because the JSON grammer is simpler!)
-var data = JSON.parse({{ data|tojson }});
+var data = {{ data }};
```

Comment:
I think we should not remove `JSON.parse` with `tojson` due to performance problems (see comment above).

Reply:
Correct. This was added for a reason. As it says in the comment directly before this line.

Reply:
Okay. I will correct it. Thanks for your vigilance.

Reply:
I added JSON.parse.